regnety/models/model.py

```diff
@@ -76,4 +79,16 @@ def RegNetY(
 
     config = get_model_config(flops)
     model = _get_model_with_config(config)
+
+    if load_weights:
+        data_url = CKPT_URL + flops + "_best.data-00000-of-00001"
+        index_url = CKPT_URL + flops + "_best.index"
+
+        ckpt_path = tf.keras.utils.get_file(origin=data_url)
+        _ = tf.keras.utils.get_file(origin=index_url)
```

Comment:
Seems like it's not being used?

Reply:
Yes, we need the file but we have the downloaded file path in `ckpt_path` itself. So there's no need to store this path again.